packages/webdriverio/src/commands/browser/switchWindow.ts

```diff
@@ -19,31 +19,31 @@
     });
  * </example>
  *
- * @param {String|RegExp}  urlOrTitleToMatch  String or regular expression that matches the title or url of the page
+ * @param {String|RegExp}  urlOrTitleOrNameToMatch  String or regular expression that matches the title or url of the page; or Name of the window
```

Comment:
How about calling this variable `matcher` rather than `urlOrTitleOrNameToMatch`, who knows what other matching patterns will be added in the future.